src/tests/dbus-tests/test_loop.py

```diff
@@ -0,0 +1,99 @@
+#!/bin/python3
+
+import unittest
+import storagedtestcase
+import subprocess
+import time
+import dbus
+import glob
+import os
+
+class StoragedLoopDeviceTest(storagedtestcase.StoragedTestCase):
+    """This is SwapSpace related functions unit test"""
+
+    LOOP_DEVICE_FILENAME = 'loop_device.img'
+
+    def get_device(self, dev_name):
+        dev = self.get_object('', '/block_devices/' + os.path.basename(dev_name))
+        return dev
+    #enddef
+
+
+    def get_loop_iface(self, device):
+        iface = dbus.Interface(device, dbus_interface=self.iface_prefix + '.Loop')
+        return iface
+    #enddef
+
+
+    def setUp(self):
+        # create a file and fill it with some data
+
+        self.FNULL = open(os.devnull, 'w')
+
+        subprocess.call(['dd', 'if=/dev/zero', 'of=%s'%self.LOOP_DEVICE_FILENAME,
+                         'bs=10MiB', 'count=1'], stdout=self.FNULL, stderr=self.FNULL)
+        # create loop device (losetup will return device name or nothing)
+        with subprocess.Popen(['losetup', '--find', '--show', self.LOOP_DEVICE_FILENAME],
+                    stdout=subprocess.PIPE, bufsize=30, universal_newlines=True) as p:
+            for line in p.stdout:
+                self.dev_name = line
+            #endfor
+        #endwith
+        assert self.dev_name != ''
```

Comment:
Please use assert methods from unittest (e.g. `self.assertNotEqual` here), instead of `assert`.